app/js/components/Draft.jsx

```diff
@@ -34,7 +33,7 @@ export class Draft extends PureComponent {
       );
 
       return (
-        <li className="draft-list small-font" key={shortid.generate()}>
```

Comment:
Ah, this to make the snapshots consistent?

Reply:
Yep!